source/samples/addressbook.d

```diff
@@ -531,20 +579,21 @@ public:
 	__gshared static SegmentReader b_98808e9832e8bc18 = GeneratedClassSupport.decodeRawBytes([
 	  0x0,0x0,0x0,0x0,0x5,0x0,0x6,0x0,
 	  0x18,0xbc,0xe8,0x32,0x98,0x8e,0x80,0x98,
-	  0x12,0x0,0x0,0x0,0x1,0x0,0x1,0x0,
+	  0x21,0x0,0x0,0x0,0x1,0x0,0x1,0x0,
```

Comment:
I don't think this is to do with the change. I believe this may have been "stale" in the existing file - is that possible?